src/coreclr/tools/aot/ILCompiler.ReadyToRun/Compiler/DependencyAnalysis/ReadyToRun/ArgIterator.cs

```diff
@@ -1651,13 +1702,13 @@ private void ForceSigWalk()
         private int _x64WindowsCurOfs;           // Current position of the stack iterator
 
         private int _armIdxGenReg;        // Next general register to be assigned a value
-        private int _armIdxStack;         // Next stack slot to be assigned a value
+        private int _armOfsStack;         // Offset of next stack slot to be assigned a value
 
         private ushort _armWFPRegs;          // Bitmask of available floating point argument registers (s0-s15/d0-d7)
         private bool _armRequires64BitAlignment; // Cached info about the current arg
 
         private int _arm64IdxGenReg;        // Next general register to be assigned a value
-        private int _arm64IdxStack;         // Next stack slot to be assigned a value
+        private int _arm64OfsStack;         // Offset of next stack slot to be assigned a value
```

Comment:
it is in byte now as well, is not it?

Reply:
Right, I've added the offset word and haven't noticed that the rest of the message contains the "slot" word.